lib/src/package.dart

```diff
@@ -1,12 +1,3 @@
-//////////////////////////////////////////////////////////////////////////////
```

Comment:
?

Reply:
It's not a generated class. This part is from old times when components were in the templates.